go/launcher/webdriver/webdriver.go

```diff
@@ -404,7 +416,7 @@ func processResponse(body io.Reader, value interface{}) (*jsonResp, error) {
 		return nil, errors.New(compName, fmt.Errorf("%v unmarshalling %q", err, respBody))
 	}
 
-	if errBody.Status == 0 && errBody.Error == "" {
+	if respBody.Status == nil || *respBody.Status == 0 {
```

Comment:
I don't understand what's happening on this line. Is this supposed to be `errBody` instead of `respBody`? And shouldn't you still be checking the `Error` field?